package-lock.json

```diff
@@ -59,8 +59,8 @@
         "typescript": "4.1.6"
       },
       "engines": {
-        "node": "16.x || 18.x || 20.x",
-        "npm": "7.x || 8.x || 9.x || 10.x"
+        "node": "18.x || 20.x",
+        "npm": "8.x || 9.x || 10.x"
```

Comment:
**Note:** @joelcarr I am guessing in v10 we didn't update all the package-locks - do you want me to include this in this change or would prefer me to take it out and only include the o-header change?

Reply:
🤦‍♂️ I did forget to do that! Yes please 🙏